### PR TITLE
Add --plugins flag to velero install

### LINKED ISSUE #1740: Add --plugins flag to `velero install` to allow plugins to be included at install time
`velero install` should allow 1+ plugins to be installed during the main install process. This would allow out-of-tree cloud providers and other plugins to be installed seamlessly, without requiring a restart.

### PR DESCRIPTION
**Pull request by nrb**
Fixes #1740

### CODE REVIEW

File: pkg/cmd/cli/install/install.go [hunk 1]
```diff
@@ -90,6 +91,7 @@ func (o *InstallOptions) BindFlags(flags *pflag.FlagSet) {
 	flags.BoolVar(&o.UseRestic, "use-restic", o.UseRestic, "create restic deployment. Optional.")
 	flags.BoolVar(&o.Wait, "wait", o.Wait, "wait for Velero deployment to be ready. Optional.")
 	flags.DurationVar(&o.DefaultResticMaintenanceFrequency, "default-restic-prune-frequency", o.DefaultResticMaintenanceFrequency, "how often 'restic prune' is run for restic repositories by default. Optional.")
+	flags.Var(&o.Plugins, "plugins", "Plugin container images to install into the Velero Deployment. Optional.")
```
Comment (carlisia) [inline pkg/cmd/cli/install/install.go]: Isn't this supposed to be required?

Comment (carlisia) [reply pkg/cmd/cli/install/install.go]: Or should I change it to required on my PR that extracts the cloudproviders?

Comment (nrb) [reply pkg/cmd/cli/install/install.go]: I can change it now if you'd like; I actually need to add a change log anyway.

It's a bit awkward either way at the moment - if it's required now, we basically have to add a dummy plugin image to get `velero install` to work on master, as the cloud provider plugins aren't present yet. 

If it's optional once the plugins are out of tree, then `velero install` could install a deployment without any object storage plugins present, which would mean it's nonfunctional.

Comment (carlisia) [reply pkg/cmd/cli/install/install.go]: Let me change it to required as part of the PR that extracts the providers then. That will only go in once we have the plugin repos and their images.

**Comment by carlisia** [comment]
Should any documentation be added now?

**Comment by carlisia** [comment]
Or, would the documentation for this be this issue: https://github.com/vmware-tanzu/velero/pull/1884?

**Comment by nrb** [comment]
I opened #1884 as an alternative to this. I think the documentation that makes sense for this PR is to document that the plugin flag exists and will install any specified container image into the deployment. Specific cloud provider instructions can be done in their respective PR(s).

**Comment by skriss** [comment]
That makes sense to me re: documentation!

Will review the updated code shortly.

File: changelogs/unreleased/1930-nrb [hunk 2]
```diff
@@ -0,0 +1 @@
+Add a new required --plugins flag for velero install command. --plugins takes a list of container images to add as initcontainers.
```
Comment (skriss) [inline changelogs/unreleased/1930-nrb]: Technically it's optional now, but will be required by the time we release - I'm fine leaving it as-is given that it'll change, but wasn't sure if it was intentional or not.

Comment (carlisia) [reply changelogs/unreleased/1930-nrb]: I'll change it to required when I merge the cloud provider extraction PR. I added a todo for that.

**Pull request merged**